scraper/avecmondoc/avecmondoc.py

```diff
@@ -0,0 +1,392 @@
+    reasons = organization["consultationReasons"]
+    if reasons is None:
+        logger.warning(f"unable to get reasons from organization {organization_id}")
+        return None
+    if not get_valid_reasons(reasons):
+        return None
+    first_availability = None
+
+    appointment_schedules = []
+    s_date = paris_tz.localize(isoparse(request.get_start_date()) + timedelta(days=0))
+    n_date = s_date + timedelta(days=CHRONODOSES["Interval"], seconds=-1)
+    appointment_schedules.append(
+        {"name": "chronodose", "from": s_date.isoformat(), "to": n_date.isoformat(), "total": 0}
+    )
+    for n in INTERVAL_SPLIT_DAYS:
+        n_date = s_date + timedelta(days=n, seconds=-1)
+        appointment_schedules.append(
+            {
+                "name": f"{n}_days",
+                "from": s_date.isoformat(),
+                "to": n_date.isoformat(),
+                "total": 0
+            }
+        )
+
+    for reason in get_valid_reasons(reasons):
+        start_date = isoparse(request.get_start_date())
+        end_date = start_date + timedelta(days=AVECMONDOC_CONF.get("slot_limit", 50))
+        request.add_vaccine_type(get_vaccine_name(reason["reason"]))
+        availabilities = get_availabilities(reason["id"], reason["organizationId"],
+            start_date, end_date, client)
+        date, appointment_count = parse_availabilities(availabilities)
+        request.appointment_count += appointment_count
+        for appointment_schedule in appointment_schedules:
+            s_date = isoparse(appointment_schedule["from"])
+            n_date = isoparse(appointment_schedule["to"])
+            name = appointment_schedule["name"]
+            if name == "chronodose" and get_vaccine_name(reason["reason"]) not in CHRONODOSES["Vaccine"]:
+                continue
+            appointment_schedule["total"] += count_appointements(availabilities, s_date, n_date)
+        request.appointment_schedules = appointment_schedules
+        if date is None:
+            continue
+        if first_availability is None or first_availability > date:
+            first_availability = date
+    
+    return(first_availability)
+
+
+def center_to_centerdict(center: CenterInfo) -> dict:
+    center_dict = {}
+    center_dict["rdv_site_web"] = center.url
+    center_dict["nom"] = center.nom
+    center_dict["type"] = DRUG_STORE
+    center_dict["business_hours"] = center.metadata["business_hours"]
+    center_dict["phone_number"] = center.metadata["phone_number"]
+    center_dict["address"] = f'{center.metadata["address"]}, {center.location.cp} {center.location.city}' 
+    center_dict["long_coor1"] = center.location.longitude
+    center_dict["lat_coor1"] = center.location.latitude
+    center_dict["com_nom"] = center.location.city
+    center_dict["com_cp"] = center.location.cp
+    center_dict["com_insee"] = departementUtils.cp_to_insee(center.location.cp)
+    center_dict["gid"] = center.internal_id
+    return center_dict
```

Comment:
Pourquoi pas simplement :

```suggestion
def center_to_centerdict(center: CenterInfo) -> dict:
    return {
        "rdv_site_web": center.url,
        "nom": center.nom,
        "type": DRUG_STORE,
        "business_hours": center.metadata["business_hours"],
        "phone_number": center.metadata["phone_number"],
        "address": f'{center.metadata["address"]}, {center.location.cp} {center.location.city}',
        "long_coor1": center.location.longitude,
        "lat_coor1": center.location.latitude,
        "com_nom": center.location.city,
        "com_cp": center.location.cp,
        "com_insee": departementUtils.cp_to_insee(center.location.cp),
        "gid": center.internal_id,
    }
```